version.go

```diff
@@ -1,4 +1,15 @@
 package specs
 
+import "fmt"
+
+const (
+	// VersionMajor is for an API incompatible changes
+	VersionMajor = 0
+	// VersionMinor is for functionality in a backwards-compatible manner
```

Comment:
On Thu, Sep 10, 2015 at 12:32:25PM -0700, Vincent Batts wrote:
> > +	// VersionMinor is for functionality in a backwards-compatible manner
> 
> this is leaning on verbiage from SemVer

Ah, consistency with semver.org is probably better than consistency
between the entries in this case.  So suggestion retracted ;).